Spec/RealtimeClientPresence.swift

```diff
@@ -136,6 +136,59 @@ class RealtimeClientPresence: QuickSpec {
                 }
             }
 
+            // RTP16
+            context("Connection state conditions") {
+
+                // RTP16c
+                pending("should result in an error if the connection state is INITIALIZED") {
```

Comment:
I'm probably missing something, but where in this test is the connection INITIALIZED? Won't it be CONNECTING right away?

Also, I'm not really sure this applies to INITIALIZED since it's not an state _change_. @mattheworiordan could you confirm this?